prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -244,13 +256,15 @@ class FHIRRouter(
             // ensure tracking is set
             actionHistory.trackCreatedReport(translateEvent, report, blobInfo)
```

Comment:
(queestion) Similar to above, do we still need to track the event if it is `NONE`?

Reply:
`trackCreatedReport` is responsible for updates to the `report_file` database table. Without performing this call, there is no record of what actions the routing step performed; it would appear as if the bundle was not routed.